inc/ui/namespace.php

```diff
@@ -71,22 +82,36 @@ function load_page() {
 	wp_enqueue_script( __NAMESPACE__, plugins_url( '/assets/script.js', Documentation\DIRECTORY . '/wp-is-dumb' ), [ 'highlightjs' ], '2019-04-19' );
 
 	// Determine the current page title.
-	$page = Documentation\get_page_by_id( get_current_group_id(), get_current_page_id() );
+	$page = Documentation\get_page_by_id( get_current_group_id(), get_current_page_id(), get_current_set_id() );
 	if ( $page ) {
 		$GLOBALS['title'] = $page->get_meta( 'title' );
 	} else {
 		$GLOBALS['title'] = __( 'Page Not Found', 'altis' );
 	}
 }
 
+/**
+ * Get the current Set ID.
+ *
+ * @return string Doc set ID if set, otherwise the default set.
+ */
+function get_current_set_id() : string {
+	// @codingStandardsIgnoreLine
+	return $_GET['set'] ?? apply_filters( 'altis.documentation.default.set', 'dev-docs' );
+}
+
 /**
  * Get the current group ID.
  *
+ * @param string $set_id The current Set id.
+ *
  * @return string Group ID if set, otherwise the default group.
  */
-function get_current_group_id() {
+function get_current_group_id( string $set_id = '' ) : string {
+	$set_id = $set_id ?? get_current_set_id();
+
 	// @codingStandardsIgnoreLine
-	return $_GET['group'] ?? 'welcome';
+	return $_GET['group'] ?? apply_filters( 'altis.documentation.default.group', 'welcome', $set_id );
```

Comment:
The function get_default_group is Altis' dev-docs function hook on to this filter. I didn't rename it because it was an already existing function. I've tried to maintain backward compat. Hence `$set_id` being added as last param with a default when logically is it the highest level and would normally be first param.